### DISCUSSION TITLE
Ender 3v3KE - KAMP

### DISCUSSION THREAD

**Discussion by simonsays10** [Guilouz/Creality-Helper-Script-Wiki#569] [Issues]
I'm trying to set up KAMP, and in the wiki for this - it shows KAMP as option 6 in improvements in the install menu. However the install menu I have is much more limited than what I see in the picture shown.  It shows 24 options on the picture, however I only see 17 on mine.  Any idea how to get the rest of it?  I should have the latest and greatest updates, I know my script is listed as 6.000 etc. on my Orcaslicer. and I'm running .13 firmware.  Not sure what other info to add, if anybody asks, I'm more than happy to provide.

**Comment by Guilouz** [comment]
All features are not available for KE series.

**Reply by simonsays10** [reply]
Ok.  I've been trying to set up KAMP but can't seem to figure out how to do that.  Can you guide me elsewhere where I can do that?

**Reply by Jaizu** [reply]
If that's not available for the KE, it should be removed from the V3 section: https://guilouz.github.io/Creality-Helper-Script-Wiki/helper-script/klipper-adaptive-meshing-and-purging/

**Reply by tabormeister** [reply]
Seems like no reason to me that the KE can't handle KAMP, it has a probe and enough processing power. I'm installing it with the KAMP install instructions, but suggest to enable this in the helper script.

**Reply by salami738** [reply]
Will this be implemented in the near future for Ender 3v3 KE?

**Reply by phatpaul** [reply]
Came to ask the same question. Why is KAMP missing from the install menu? I want to know if it Is it kept off for valid reason Before I try to install it manually using the instructions here: https://github.com/kyleisah/Klipper-Adaptive-Meshing-Purging
Thanks!

**Reply by Hogan1975** [reply]
I've got KAMP working fine on two KE's.  Go to https://github.com/kyleisah/Klipper-Adaptive-Meshing-Purging  to install but change commands for installing.   Do the first command as normal to Clone from github to root of drive.  The next two commands are different  because of the location difference in Creality OS version of Klipper vs Vanilla Klipper.  You still need to add exclude object in printer.cfg and [file_manager] enable_object_processing: True in the Moonraker.conf.

Then Send this command:
ln -s /root/Klipper-Adaptive-Meshing-Purging/Configuration /usr/data/printer_data/config/KAMP

Then Send this command:
cp /root/Klipper-Adaptive-Meshing-Purging/Configuration/KAMP_Settings.cfg /usr/data/printer_data/config/KAMP_Settings.cfg


Check out Google AI information on different commands to install through the putty software....

https://www.google.com/search?q=ender+3+v3+KE+ln+-s+%7E%2FKlipper-Adaptive-Meshing-Purging%2FConfiguration+printer_data%2Fconfig%2FKAMP&sca_esv=76156e36b6817723&rlz=1C1RXQR_enUS1100US1121&sxsrf=ANbL-n5pMxa-nKKUXAw0TjUdm9D35uk0lQ%3A1769219669618&udm=50&fbs=ADc_l-aN0CWEZBOHjofHoaMMDiKp9lEhFAN_4ain3HSNQWw-mI4C1ELY88XaJ57aZGHEMB0-yG_rbcGKCggd0y127_8yS-tF29BKbhPeQOXvV6o-X4A9ZRgYWyxhD2V5oSWzSAqmOSymlIg7h1v8S9Rw50Gwhe5ca5eQd3lypdEGp_J0RPTGdHo04gpbja8CH3sRJUYhXSzk2gSttlJweKXFGiy4OOo9xg&aep=1&ntc=1&sa=X&ved=2ahUKEwiTsr_viKOSAxUf1fACHcGfBssQ2J8OegQIDxAE&biw=1536&bih=695&dpr=1.25&aic=0&mtid=WyZ0abKlFtiW5OMPt9DV0Qs&mstk=AUtExfBHX5xGWjTQMW78aC4t92yIQ3-k3ywnHxjP_QeN7aiQp-ejm036PcalHTz4bITBOIxyw2gtYgNZ8dOa7b8Bk-JuyZLzimIztv2kyADteEtcG7nofneSIoS_BP3X-Azafy-WSz_TKBGmBVWyn2ukI9RShH3gIoZM1VTSrn99M7KEtGP-BgFXYTI2SobnPJCFBPvlm6iuqsYDqsO1qAbzVJaYaCc6ZKJGlHR5Wj4iVH97ZBDUU8TDsP1qEWlGeusv4EHX7SZgoo6QKa9A50h76AuIfVMT06o988F7ZF3W7ciCwlCQ4kBzWda0W71HB-K7Nk1LylqlP13IwBkuBZaFWuO4jCU4MR6a--hZvZzOKQeyON6NGYFEzU1rrurxVLTbNGpN95NrOv-MZQlQnU_zZpdqYt4hQjXgUdfEu51oQabi8NF7dVXaX9t8-mJuLcwSy37SIZfx6F20ABXG66RpqN4hCmZ4ypM3ws0YETA0XFAJqKsuQzRRG7I5WBvBYILYo91ams2U3U6vHSCLAj8WwF3aPJlh9Kk&csuir=1

**Reply by DeNohvo** [reply]
is it still working good?

**Reply by Hogan1975** [reply]
Mines been working great.

**Comment by DeNohvo** [comment]
any reason why this is missing?

**Comment by tabormeister** [comment]
I realized with Orca adaptive bed meshing there was no reason to install Kamp so I didn't

**Reply by Hogan1975** [reply]
I tried the Orca built-in method of adaptive meshing.  It lacks the adaptive purging that KAMP does.

**Reply by tabormeister** [reply]
nah there's orca gcode for that too, I have mine do an adaptive lil line 20mm below the outside left of the print.
https://www.printables.com/model/1035759-adaptive-purge-for-any-3d-printer-using-slicer-var

**Reply by Hogan1975** [reply]
Glad it's working for ya
